ddl/callback.go

```diff
@@ -28,6 +33,8 @@ type Callback interface {
 	OnJobUpdated(job *model.Job)
 	// OnWatched is called after watching owner is completed.
 	OnWatched(ctx context.Context)
+	// OnGetInfoSchema is a hook which is called in the function ddl.GetInfoSchema(). It is used in the tests.
+	OnGetInfoSchema(ctx sessionctx.Context, fn GetInfoSchema) infoschema.InfoSchema
```

Comment:
I think this function doesn't belong to the callback.
It's actually an intercepter.